tests/redirects/map_globalconf.py

```diff
@@ -1100,4 +1100,7 @@
     url_test('/firefox/{49.0,49.0.1,50.0a1,51.0a2}/firstrun/learnmore', '/firefox/features/'),
     url_test('/firefox/windows-10/welcome', 'https://support.mozilla.org/kb/how-change-your-default-browser-windows-10'),
 
+
+    # Bug 682619
+    url_test('/support/thunderbird', 'https://support.mozilla.org/en-US/products/thunderbird'),
```

Comment:
You might want to move this test up to accompany the test for the existing redirect too, rather than leave them separated.